src/SearchInput/SearchInput.tsx

```diff
@@ -81,27 +96,22 @@ export const SearchInput = forwardRef<HTMLInputElement, SearchInputProps>(
     const [searchQuery, setSearchQuery] = useState<string | null>(null)
 
     const fuse = useMemo(() => {
-      const searchKeys = ['label', 'value']
-      return new Fuse(searchList, {
-        keys: searchKeys,
-        findAllMatches: true,
-        minMatchCharLength: 2,
-      })
+      return new Fuse(searchList, fuzzySearchOptions)
     }, [searchList])
 
     const filteredList = useMemo(() => {
       if (searchQuery === null || searchQuery === '') {
         return searchList
       }
 
-      if (enableFuzzySearch) {
+      if (enableFuzzySearch || !!fuzzySearchOptions) {
```

Comment:
added this here - so we're not relying on two props for this to work, unsure how people feel about this, but i did spend 20mins trying to work out why this wasn't working then realised i didnt have enableFuzzySearch as true

Reply:
Because `fuzzySearchOptions` is provided as a default in this component, this would always be true. Imo we would only need `enableFuzzySearch` as the variable here.